scripts/support_bundle/README.md

```diff
@@ -0,0 +1,39 @@
+# Support Bundle Generation Script
+
+This script is to be used to collect information on your Cloud Connector Scale Set Deployment Zscaler Customer Support. The script will collect the health and scale metrics published by the Cloud Connectors, the configuration of the Scale Set deployment, and the logs from the Function App. This bundle will be used for debugging purposes to help support customer.
+
+## Running Script
+```
+cd scripts/support_bundle
+python -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+python generate_support_bundle.py --resource_group <resource-group> --time_duration <time-in-min> --subscription_id <subscription-id> --tenant_id <(optional)tenant-id> --client_id <(optional)client-id> --client_secret <(optional)client-secret>
+```
+
+1. Navigate to the support_bundle directory
+2. Setup virtual environment to run the script in
+3. activate the virutal environment
```

Comment:
just a typo on virtual

Reply:
Will adress this in upcoming PR with permission information